app/assets/stylesheets/modules/org.css

```diff
@@ -1,17 +1,17 @@
 /* Flash Intercoms */
 
 .flash {
-  border-bottom: 1px solid #e9573f;
+  border-bottom: 1px solid #e9573f45;
   background-color: white; }
 
 .flash-wrap {
   padding-top: 5px;
   padding-bottom: 5px;
-  background-image: linear-gradient(to right, rgba(233, 87, 63, 0.3), #e9573f); }
+  padding: 8px 0px;
```

Comment:
A few notes:
* The `padding-top` & `padding-bottom` rules are redundant. Let's delete them. The `padding` rule on Line 10 overwrites the rules above.
* Can we increase the top/bottom padding by a bit more, increase to `10px`
* For the `0px` left/right padding, drop the `px` unit. Units aren't required when the value is `0`.

```css
.flash-wrap {
  padding: 10px 0;
  background-color: #fff6d2;
}
```